apps/andi/assets/css/edit.scss

```diff
@@ -35,6 +59,29 @@
   color: $color-success;
 }
 
+#test-status {
```

Comment:
Using ids feels wrong to me but I won't fight in that war.

Reply:
They were used elsewhere so I continued the trend. Is there a problem using ids to identify styles for unique elements on a page?

I assume that BEM does not provide for using ids like this.